datadog_checks_dev/datadog_checks/dev/tooling/commands/agent/changelog.py

```diff
@@ -66,14 +66,7 @@ def changelog(since, to, write, force):
             for entry in CHANGELOG_MANUAL_ENTRIES.get(agent, []):
                 changelog_contents.write(f'{entry}\n')
             for name, ver in version_changes.items():
-                # get the "display name" for the check
-                manifest_file = os.path.join(get_root(), name, 'manifest.json')
-                if os.path.exists(manifest_file):
-                    decoded = json.loads(read_file(manifest_file).strip())
-                    display_name = decoded.get('display_name')
-                else:
-                    display_name = name
-
+                display_name = get_display_name(name)
```

Comment:
I think we should use the `manifest.get_display_name`  thats defined here - https://github.com/DataDog/integrations-core/blob/27245b33541e64e6c61b0eb78fe323de5e638ce9/datadog_checks_dev/datadog_checks/dev/tooling/manifest_utils.py#L139-L140

IMO that global helper should be deprecated in favor of Manifest specific getters (to better set us up for potential future manifest versions)

Reply:
This function only works for manifest V2 though, correct?

Reply:
If you load the manifest with the helper I shared, you get back either a V1 or V2 class depending on the version of the manifest. Both have a `get_display_name` method defined, to try and help abstract the actual lookup.